### ISSUE TITLE
13 pro max 15.0.1 apps just dont do anything when shared

### ISSUE THREAD

**Issue by troppp** [opa334/TrollStore#7]
After sharing an app to troll store (I've tried it from safari, on my iphone files, google drive) it just doesn't do anything. I have ldid installed and everything it just doesn't do the thing lol. seems dope tho.

**Comment by opa334** [comment]
installing apps from icloud drive currently isn't working, will look into it soon, for now copy them to local storage and then it should work

**Comment by troppp** [comment]
Yes I copied them to local storage and tried many different storage apps it still doesn't do anything... I can send a video if that's helpful?

**Comment by opa334** [comment]
the only thing I can think of is you already have an app with the same identifier installed on your device, in that case TrollStore will refuse to install it

**Comment by troppp** [comment]
I tried it with Delta (which I don't have installed anywhere) and it still doesn't go idk someone else in r/jb discord was having the same issue where it just doesn't work.

**Comment by troppp** [comment]
they said they tried many different ipas too

**Comment by opa334** [comment]
no idea what's the issue, do you have access to a Mac? in console.app go to your device in the left (needs to be attached via USB), then enter "trollstorehelper" in the top right search bar while it's logging. Then try to install an ipa and send me the log (CMD+A, CMD+C).

**Comment by troppp** [comment]
Alright here is the output:

```
default    17:42:43.128123-0500    kernel    App Store Fast Path -> /private/var/containers/Bundle/Application/8C6F6303-C80D-4311-81A4-8B559EEDCF9D/TrollStore.app/trollstorehelper
default    17:42:43.136931-0500    trollstorehelper    trollstore helper go, uid: 0, gid: 0
default    17:42:43.142833-0500    trollstorehelper    error: Error Domain=NSCocoaErrorDomain Code=260 "The folder “Payload” doesn’t exist." UserInfo={NSUserStringVariant=(
    Folder
), NSFilePath=/var/tmp/40B15939-F868-435F-BDBF-83028785B8B9/Payload, NSUnderlyingError=0x14d3073d0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}} 
```

**Comment by opa334** [comment]
I'm 90% sure that's the iCloud drive issue, but I'm looking into fixing it now

**Comment by troppp** [comment]
Okay cool, this was me trying an ipa from safari

**Comment by opa334** [comment]
should be fixed in 1.0.1, download TrollStore.tar and open it in TrollStore to update

**Comment by troppp** [comment]
It works now but I was actually unable to share the tar into troll store. It appears I was unable to open anything into TrollStore. It was not in iCloud. But thanks so much for fixing it works well.
